### PR TITLE
Update to M121

### PR DESCRIPTION
**Pull request by trevorsg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,14 +26,16 @@
...
   "types": "./typings/index.d.ts",
   "dependencies": {
-    "@types/jquery": "^2.0.34",
-    "@types/jqueryui": "^1.11.31",
-    "@types/knockout": "^3.4.38",
+    "@types/jquery": ">=2.0.34",
+    "@types/jqueryui": ">=1.11.31",
+    "@types/knockout": ">=3.4.38",
+    "@types/knockout-secure-binding": ">=0.0.28",
+    "@types/mousetrap": ">=1.5.0",
```
Comment (cschleiden) [inline package.json]: are we exposing that now?

Comment (trevorsg) [inline package.json]: Are you referring to mousetrap? If so, yeah, not much of a choice. Would love to not have this dependency though...

Comment (cschleiden) [inline package.json]: Yes... don't think we're exposing any functionality

Comment (serkan-inci) [inline package.json]: It would be great if we could get rid of that dependency. I'll see what would it take to get rid of these dependencies.

Comment (trevorsg) [inline package.json]: @serkan-inci @cschleiden Fixed :)

**Pull request merged**